src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -85,12 +85,17 @@ public static function debug_data() {
 			'wp-plugins-active'   => array(),
 			'wp-plugins-inactive' => array(),
 			'wp-media'            => array(),
-			'wp-server'           => array(),
+			'wp-server'           => self::get_wp_server(),
 			'wp-database'         => self::get_wp_database(),
 			'wp-constants'        => self::get_wp_constants(),
 			'wp-filesystem'       => self::get_wp_filesystem(),
 		);
 
+		// Remove accordion for Directories and Sizes if in Multisite.
+		if ( $is_multisite ) {
+			unset( $info['wp-paths-sizes'] );
```

Comment:
@dmsnell See lines 167-173 and 288-348, `$info['wp-paths-sizes']` shall only be visible on single sites, so I've added this unset here. I think that was why I originally opted for `$info['key'] = self::get_key;` instead of your approach. 

And I've knowingly used `is_multisite()` instead of `$is_multisite`, we will remove `$is_multisite` later in the refactoring, since it is likely only helpful in the upcoming `function get_wp_core()`

Reply:
this seemms fine. thanks @apermo

Reply:
I've updated the comment in f576b043ae

Reply:
we could eventually filter this array and have `self::get_wp_paths_sizes()` return `null` for multi-site deployments.